### DISCUSSION TITLE
skywalking support for spring-boot-data-cassandra 3.3.1

### DISCUSSION THREAD

**Discussion by abhinandc123** [apache/skywalking#11989] [Q&A]
how do develop one plugin spring-boot-data-cassandra 3.3.1.How can i identify which classes and method  to intercept . github link to driver repo - https://github.com/spring-projects/spring-data-cassandra/tree/3.3.x

**Comment by wu-sheng** [comment]
Read the source codes, there is no magic to tell which classes.
